charts/aws-ebs-csi-driver/templates/csidriver.yaml

```diff
@@ -10,3 +11,4 @@ spec:
   {{- if not .Values.useOldCSIDriver }}
   fsGroupPolicy: File
   {{- end }}
+{{- end }}
```

Comment:
If there is a requirement to skip installing this, `disableController` might not be the appropriate knob for it.

The CSIDriver object should still be deployed even when the controller component is disabled. The developer [docs](https://kubernetes-csi.github.io/docs/csi-driver-object.html#csidriver-object) mention its used to specify how Kubernetes should interact with a driver (not just in the context of the controller plugin).

Reply:
If the parameter was renamed to `NodeDaemonsetOnly` would that change your answer? 

> The CSIDriver object should still be deployed even when the controller component is disabled.

I'm trying to prevent the case where there is a clash because the CSIDriver resource (along with controller and all other resources) is already deployed.

Consider the following workflow:

1. Install driver (Controller + a few special nodes) 
2. Install driver in NodeOnly mode

If we don't avoid deploying csidriver object in either of the two, there would be a clash, right?